pkg/operatorhub/operatorhub.go

```diff
@@ -46,6 +47,21 @@ func (o *operatorhub) Get() map[string]bool {
 	return o.current
 }
 
+// Disabled returns true if all defaults are disabled
+func (o *operatorhub) Disabled() bool {
+	o.lock.Lock()
+	defer o.lock.Unlock()
+
+	disabled := true
```

Comment:
This function always returns true, I think you want:
```go 
o.lock.Lock()
defer o.lock.Unlock()

for _, disabled := range o.current {
  if disabled == true {
    return false
  }
}
return true
```

(there should probably be a test for this as well)

Reply:
sorry this isn't clear because i named two variables the same thing -- this is intending to check that *all* of the values in that map are set to false.

That's because our choice is either:

1. Some (or all) of the defaults are enabled. In that case, just do the normal behavior. Something will reconcile and we should ensure it's working and report when it is not.
2. NONE of the defaults are enabled, in which case just say that we are available.

So to make this more readable it should look something like:

``` go
o.lock.Lock()
defer o.lock.Unlock()

allDisabled = true
for _, disabled := range o.current {
  if disabled == false {
    allDisabled = false
  }
}

return allDisabled
```